app/main.py

```diff
@@ -1,3 +1,63 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.Shop import Shop
+from app.Car import Car
+from app.Customer import Customer
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers_data = data["customers"]
+    shops_data = data["shops"]
+
+    shops = []
+    for shop_info in shops_data:
+        shop_name = shop_info["name"]
+        shop_location = shop_info["location"]
+        products = shop_info["products"]
+        shop = Shop(shop_name, shop_location, products)
+        shops.append(shop)
+
+    for customer_data in customers_data:
+        customer_car = Car(
+            customer_data["car"]["brand"],
+            customer_data["car"]["fuel_consumption"])
```

Comment:
n case you leave a bracket on a line without arguments, your closing bracket should not have arguments on the same line as well.